packages/nimble-components/src/rich-text-mention/base/models/mapping-user-config.ts

```diff
@@ -0,0 +1,14 @@
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { ListOption, listOptionTag } from '../../../list-option';
+import { MappingConfig } from './mapping-config';
+
+/**
+ * Mapping configuration corresponding to a user mapping
+ */
+export class MappingUserConfig extends MappingConfig {
+    public override createListView(): ViewTemplate<ListOption> {
+        return html`<${listOptionTag} value="${this.mentionHref ?? ''}">${
+            this.displayName ?? ''
+        }</${listOptionTag}>`;
+    }
```

Comment:
I'm a bit concerned about this. Firstly we should not dynamically construct templates. They should be statically defined, usually by being in a separate file and imported. 

I can't provide a more concrete alternative without knowing more about the use case though. Is the concern about how flexible the `@mention` drop down picker is? I think initially we can keep it simple and say it's list options for all mention types, and the only state needed is the mentionHref and displayHref which is in the MappingConfig base class. MappingUserConfig should just be empty.

```
MappingUserConfig extends MappingConfig {}
```

Reply:
Removed `createListView()` method and only having `mentionHref` and `displayName` in base class.